sky/authentication.py

```diff
@@ -67,6 +64,13 @@
 # the former dir is empheral.
 _SSH_KEY_PATH_PREFIX = '~/.sky/clients/{user_hash}/ssh'
 
+if typing.TYPE_CHECKING:
+    import filelock
+    import yaml
+else:
+    filelock = adaptors_common.LazyImport('filelock')
+    yaml = adaptors_common.LazyImport('yaml')
```

Comment:
would be good to mention the time scale of the imports here in the comments, so that future maintainance can use them as a reference

Reply:
Do you mean to comment the time of non-lazy importing `filelock&yaml` here or the time of importing `authentication` after this change?

Reply:
I have added comments for the third-party modules in `sky/adaptors/common.py` and descriptions in the contributing guide.